src/components/dropdown-menu/DropdownMenuItemLink.tsx

```diff
@@ -19,18 +19,20 @@
  */
 
 import styled from '@emotion/styled';
+import * as VisuallyHidden from '@radix-ui/react-visually-hidden';
 import { forwardRef } from 'react';
+import { useIntl } from 'react-intl';
+import { NavLink as RouterNavLink, NavLinkProps as RouterNavLinkProps } from 'react-router-dom';
+import { isSonarLink } from '~common/helpers/url';
 import { IconLinkExternal } from '../icons/IconLinkExternal';
-import { LinkBase, LinkBaseProps } from '../links/LinkBase';
 import { DropdownMenuItemBase, DropdownMenuItemBaseProps } from './DropdownMenuItemBase';
 
 type Props = Omit<
   DropdownMenuItemBaseProps,
   'isCheckable' | 'isChecked' | 'ItemWrapper' | 'itemWrapperProps'
-> &
-  Pick<
-    LinkBaseProps,
-    'download' | 'hasExternalIcon' | 'isMatchingFullPath' | 'shouldOpenInNewTab' | 'to'
+> & { hasExternalIcon?: boolean } & Pick<
+    DropdownMenuItemNavLinkProps,
```

Comment:
I would have expected to see `DropdownMenuItemNavLinkProps` defined before it is used - maybe move the type(s) up?

Reply:
I'll actually move the navlink outside this folder.